todo_app/app.py

```diff
@@ -6,71 +6,71 @@
 from todo_app.data.trello import TrelloUtility
 from todo_app.data.model import ViewModel
 
+from dotenv import load_dotenv, find_dotenv
+
 import os
 
 
+app = Flask(__name__)
 
-def create_app():
```

Comment:
Moving everything out of `create_app` will break your tests. Undo this change

Reply:
This has been added back in